the pl comment should go to .po file really

--- gtkpod-2.1.0/data/gtkpod.desktop.in~	2011-08-06 19:03:49.000000000 +0300
+++ gtkpod-2.1.0/data/gtkpod.desktop.in	2011-08-15 16:58:56.946994824 +0300
@@ -2,9 +2,11 @@
 _Name=gtkpod
 _GenericName=iPod Manager
 _Comment=Manage music and video on an Apple iPod
+# TODO: use add po/pl.po create pl.po-update.patch or sth, and submit it upstream
+Comment[pl]=Graficzny menadżer utworów muzycznych dla urządzeń Apple iPod
 Exec=gtkpod
 Icon=gtkpod
 Terminal=false
 Type=Application
-Categories=GTK;AudioVideo;
+Categories=GTK;Utility;Audio;
 X-GNOME-FullName=gtkpod iPod Manager
