--- gtkpod-2.1.0/./CMakeLists.txt~	2011-07-17 20:46:59.000000000 +0300
+++ gtkpod-2.1.0/./CMakeLists.txt	2011-08-18 01:19:43.993660139 +0300
@@ -51,7 +51,7 @@
 # typedef void* MP4FileHandle;//>
 # #include <mp4v2/itmf_tags.h>
 # This matches the way it's used in the code, and the autoconf test
-CHECK_INCLUDE_FILES("mp4v2/platform.h>\ntypedef void* MP4FileHandle\;//;mp4v2/itmf_tags.h" HAVE_MP4V2_ITMF_TAGS_H)
+CHECK_INCLUDE_FILES("mp4v2/platform.h>\ninclude <stdint.h>\ntypedef void* MP4FileHandle\;//;mp4v2/itmf_tags.h" HAVE_MP4V2_ITMF_TAGS_H)
 
 set(OPT_INCLUDES "")
 set(OPT_CFLAGS "")
--- gtkpod-2.1.0/configure.ac	2011-08-18 01:19:58.317693619 +0300
+++ gtkpod-2.1.0/configure.ac	2011-08-18 01:24:48.655205091 +0300
@@ -280,9 +280,12 @@
 	dnl rollback to older mp4 library - should help with distros
 	dnl not using 1.9.1 of the mp4v2 library
 	AC_CHECK_HEADERS([mp4.h], have_mp4=yes, have_mp4=no)
+else
+	AC_CHECK_HEADERS([mp4v2/platform.h], [], [])
 fi
 AC_CHECK_HEADERS([mp4v2/itmf_tags.h], [], [],
-		 [#ifdef HAVE_MP4V2_PLATFORM_H
+		 [#include <stdint.h>
+		  #ifdef HAVE_MP4V2_PLATFORM_H
 		  # include <mp4v2/platform.h>
 		  #endif
 		  typedef void* MP4FileHandle;
