diff -uNr gtkpod-2.1.3.old/src/gtkpod.c gtkpod-2.1.3/src/gtkpod.c
--- gtkpod-2.1.3.old/src/gtkpod.c	2012-05-11 23:25:00.000000000 +0200
+++ gtkpod-2.1.3/src/gtkpod.c	2013-03-30 22:39:47.623499920 +0100
@@ -29,6 +29,7 @@
 #include <gtk/gtk.h>
 #include <libanjuta/anjuta-utils.h>
 #include <libanjuta/anjuta-shell.h>
+#include <libanjuta/anjuta-version.h>
 
 #include "gtkpod.h"
 #include "libgtkpod/directories.h"
@@ -236,7 +237,10 @@
     anjuta_shell_session_save(ANJUTA_SHELL(app), session_dir, NULL);
     g_free(session_dir);
 
+#if ANJUTA_CHECK_VERSION(3,7,3)
+#else
     anjuta_shell_notify_exit(ANJUTA_SHELL(app), NULL);
+#endif
 
     if (!gtkpod_cleanup_quit()) {
         // Dont want to quit so avoid signalling any destroy event
